src/app/recurring-deposits/recurring-deposits-account-view/recurring-deposits-account-view.component.html

```diff
@@ -0,0 +1,202 @@
+            </tr>
+            <tr>
+              <td>Period</td>
+              <td>
+                <span>{{recurringDepositsAccountData.depositPeriod}}&nbsp;{{recurringDepositsAccountData.depositPeriodFrequency.value}}</span>
+              </td>
+            </tr>
+            <tr>
+              <td>Deposit Frequency</td>
+              <td><span>{{recurringDepositsAccountData.recurringFrequency}}
+                  &nbsp;{{recurringDepositsAccountData.recurringFrequencyType.code}}
+                </span></td>
+            </tr>
+            <tr *ngIf="recurringDepositsAccountData.minBalanceForInterestCalculation">
+              <td> Balance Required For Interest Calculation</td>
+              <td><span>{{recurringDepositsAccountData.minBalanceForInterestCalculation}}</span></td>
+            </tr>
+            <tr *ngIf="recurringDepositsAccountData.witdHoldTax">
+              <td> Witdhold Tax Group </td>
+              <td><span>{{recurringDepositsAccountData.taxGroup.name}}</span></td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+
+      <div fxFlex="49%"
+        *ngIf="!recurringDepositsAccountData.status.rejected && !recurringDepositsAccountData.status.submittedAndPendingApproval">
+        <h4 class="table-headers">Performance History</h4>
+        <table>
+          <tbody>
+            <tr>
+              <td>Principal Amount</td>
+              <td>
+                <span>{{recurringDepositsAccountData.currency.displaySymbol}}&nbsp;{{recurringDepositsAccountData.depositAmount}}</span>
+              </td>
+            </tr>
+            <tr>
+              <td>Maturity Amount</td>
+              <td>
+                <span>{{recurringDepositsAccountData.currency.displaySymbol}}&nbsp;{{recurringDepositsAccountData.maturityAmount | number}}</span>
+              </td>
+            </tr>
+            <tr>
+              <td>Recurring Deposit Amount</td>
+              <td>
+                <span>{{recurringDepositsAccountData.currency.displaySymbol}}&nbsp;{{recurringDepositsAccountData.mandatoryRecommendedDepositAmount | number}}</span>
+              </td>
+            </tr>
+            <tr>
+              <td>Deposits till Date</td>
+              <td><span
+                  *ngIf="recurringDepositsAccountData.summary.totalDeposits">{{recurringDepositsAccountData.currency.displaySymbol}}&nbsp;{{recurringDepositsAccountData.summary.totalDeposits | number}}</span>
+                <span
+                  *ngIf="!recurringDepositsAccountData.summary.totalDeposits">{{recurringDepositsAccountData.currency.displaySymbol}}&nbsp;0</span>
+              </td>
+            </tr>
+            <tr>
+              <td>Withdrwals till Date</td>
+              <td><span
+                  *ngIf="recurringDepositsAccountData.summary.totalWithdrawals">{{recurringDepositsAccountData.currency.displaySymbol}}&nbsp;{{recurringDepositsAccountData.summary.totalWithdrawals | number}}</span>
```

Comment:
single ngIf on tr will do

Reply:
I know.. that's a better way of implementation.